RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -73,6 +73,11 @@ boolean EightBitRealmHandler()
 	boolean adv_spent = false;
 
 	string color = get_property("8BitColor");
+	if((internalQuestStatus("questL02Larva") < 0 && internalQuestStatus("questG02Whitecastle") < 0) && available_amount($item[Continuum Transfunctioner]) == 0)
```

Comment:
did you run into an issue with this? Curious about the story behind this change

Reply:
I did run into an issue with this. Basically autoscend tried to access the 8-bit realm before it had completed the Woods quest. I tried to add woods_questStart initially as a fix but that didn't work so I added a combination of the conditionals in woods_questStart to this (minus the KoE one because KoE will never hit this). This change prevents trying to go to the 8-bit realm until the distant woods are unlocked and we have a continuum transfunctioner